common/distribution/aciarchive.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2016 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package distribution
+
+import (
+	"fmt"
+	"net/url"
+)
+
+const DistACIArchiveVersion = 0
+
+// ACIArchive defines a distribution using an ACI file
+// The format is:
+// cmd:aci-archive:v=0:ArchiveURL?query...
+// The distribution type is "archive"
+// ArchiveURL must be query escaped
+// Examples:
+// cimd:aci-archive:v=0:file%3A%2F%2Fabsolute%2Fpath%2Fto%2Ffile
+// cimd:aci-archive:v=0:https%3A%2F%2Fexample.com%2Fapp.aci
+type ACIArchive struct {
```

Comment:
Good question. I had the same though when designing it. Looking at rkt you can pass the image signature with `--signature` as an additional option for http, file etc... and this can also be used to override the appc discovered signature.

So I thought to leave it outside distribution because I'd also have to find a way to specify an ovverride signature for appc distribution and that it will generate different distributions for every different passed signature.

I'm not completely sure that my reasoning is correct so I'd like to hear you thoughts!